RhysMP.md

```diff
@@ -0,0 +1,37 @@
+---
+name: "@RhysMP"
+project: "DonkeyCar RC car"
+---
+
+# DonkeyCar RC car
+
+## Summary
+
+I plan on building a RC car which can  drive itself and also be controlled from mobile. I'm super excited as I have always wanted to experiemnt with a rasberry pi and see what amazing things I could do with it. I also have always thought RC cars were fasinating. 
+
+## Plan
+
+First I will order all of the products. 
+
+Then I will send all the CAD files required for this car to my HS's Physics teacher to 3D print using the makerbots we have on campus.
+
+I will then clean up all of these prints.
+
+I will attach the parts together. (Pi > 3d printed plate) (Camera to Adatper)
+
+Then I will install the Donkeycare software and configure it all.
+
+http://docs.donkeycar.com/
+
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Red LaTrax® Desert Prerunner: 1/18-Scale 4WD Electric Truck       | https://latrax.com/products/prerunner | 142.30 (Includes tax) |
```

Comment:
Do you think you could build your own car instead of buying one?

It looks like the Donkey Car documentation has [a good guide on doing so](http://docs.donkeycar.com/guide/build_hardware/). I think building your own car would make your project proposal more compelling as it incorporates more "building"

Reply:
I think I would perfer using the prebuilt one. Looking at it is is better supported documentation wise and I don't think "rolling a car" as described in the links above is within my skill level. I think it would also be more expensive.

During this I would want to code custom firmware or do further modifications. ESPEICALLY on the software side coming from a CS background

Reply:
Interesting, could you elaborate more on the custom firmware? What will you use to build the firmware; what kind of customizations will you make?

Reply:
Donkey has there own software but I would like to code my own. Having recently taken Computer Architecture I've learned about lower level programming and interfacing with electronics. I would want to be able to control every aspect about the RC from my computer and be able to custom code routines for it to be able to follow if possible using the embedded camera.

Reply:
@exu3

Reply:
Ok, that sounds great!

Reply:
Could you add that to your proposal?